Modules/Segmentation/LevelSets/include/itkParallelSparseFieldLevelSetImageFilter.h

```diff
@@ -301,7 +301,11 @@ class ITK_TEMPLATE_EXPORT ParallelSparseFieldLevelSetImageFilter
   using StatusType = signed char;
 
   /** Type used for storing the number of layers */
-  using LayerCountType = signed char; /* purposefully small integer same as StatusType for backward compatibility */
+  using LayerCountType = StatusType; /* purposefully the same as StatusType
```

Comment:
The commit message does not correspond to this change.

Reply:
OH:  commit message.... not "comment message".  Perhaps I did a rebase squash wrong.